src/output.rs

```diff
@@ -251,3 +251,42 @@ impl Output for Status {
         Ok(Status(result?.exit_status))
     }
 }
+
+/// Using [`bool`] as the return type for [`cmd!`] will return `true` if
+/// the command returned successfully, and `false` otherwise:
+///
+/// ```
+/// use cradle::*;
+///
+/// if !cmd!(%"echo foo") {
+///     panic!("echo failed!");
+/// }
+/// ```
+///
+/// Also, when using [`bool`], non-zero exit codes will not result in a panic
+/// or [`std::result::Result::Err`]:
+///
+/// ```
+/// use cradle::*;
+///
+/// let success: bool = cmd!("false");
+/// assert!(!success);
+/// let result: Result<bool, cradle::Error> = cmd_result!("false");
+/// assert!(result.is_ok());
+/// assert!(!result.unwrap());
```

Comment:
```suggestion
/// assert_eq!(result, Some(false));
```

I _think_ this is slightly better. TIOLI.